_includes/apple-touch-startup-image.html

```diff
@@ -20,4 +20,4 @@
 {%- endcapture -%}
 <link rel="apple-touch-startup-image"
     media="{{ device_width_string }} and {{ device_height_string }} and {{ device_orientation_string }} {{ pixel_ratio_string }}"
-    href="{{ design_guide_url }}/icons/apple-touch-startup-image-{{- image_size -}}.png">
+    href="{{ design_guide_url }}//icons/apple-touch-startup-image-{{- image_size -}}.png">
```

Comment:
Should it be a double slash here?

```suggestion
    href="{{ design_guide_url }}/icons/apple-touch-startup-image-{{- image_size -}}.png">
```

Reply:
It is actually a double slash there according to https://design.swedbankpay.com/v/7.1.0/identity/logotype

Reply:
Weird. @VikingTristan @PhungNg @eskilsen is this correct?

Reply:
I believe it works because the browser understands that any number of slashes should just be _one_ slash in a url.

E.g. this works:
![image](https://user-images.githubusercontent.com/26485094/144843685-d9cfe402-031d-41e0-9a19-35d7934ecde3.png)

Reply:
Aha. So it's just me that don't understand that the browser understands!